packages/integration-karma/test/swapping/styles/index.spec.js

```diff
@@ -25,4 +25,14 @@ describe('style swapping', () => {
                 );
             });
     });
+
+    afterEach(() => {
+        // FIXME: Currently, hot swapping does not actually remove styles from the DOM
+        for (const elm of document.head.querySelectorAll('style')) {
+            elm.parentElement.removeChild(elm);
+        }
+        if (document.adoptedStyleSheets) {
+            document.adoptedStyleSheets = [];
+        }
+    });
```

Comment:
It would be great to fix this, but it only happens in dev mode, and hot-swapping is not even used on platform.